templates/adding_a_new_model/cookiecutter-template-{{cookiecutter.modelname}}/modeling_tf_{{cookiecutter.lowercase_modelname}}.py

```diff
@@ -1151,7 +1249,7 @@ def call(
             training=training,
             kwargs_call=kwargs,
         )
-        outputs = self.{{cookiecutter.uppercase_modelname}}(
+        outputs = self.{{cookiecutter.lowercase_modelname}}(
```

Comment:
@jplu this was the issue that made the model templates crash; fixed it here and put the tests back on.

Reply:
Awesome you found it!!